docs/data-provider/README.md

```diff
@@ -0,0 +1,41 @@
+# Data Provider Page
+
+This page consists of the following sections and features:
+
+### Hero
+
+This section allows user to contribute to TDC by redirecting to the `dashboard/datasets/create` page. And listing the Organizations that have already provider data to TDC.
+
+![Hero section](hero.png)
+
+### How TDC datasets work
+
+This section provides an overview of different types of datasets in TDC.
+
+![TDC Datasets](types.png)
+
+### Add Data Section
+
+![Add Data](add_data.png)
+
+### How To Add Data
+
+![How To](how_to.png)
+
+#### Featured Asset
+
+Allows the configuration of the element that appears on this section, which can be an **image**, an **mp4 video**, or a **YouTube embedded video**.
+**How to configure**: In the `public` folder of the frontend project, the following properties should be set in the `tdc-config.json` file to define the type and file path to be displayed:
+
+```
+{
+  "hero": {
```

Comment:
if its supposed to use the same as Home page you should implement the video section as its in Home page. maybe create a component